docs/source/public/overview.rst

```diff
@@ -733,12 +735,21 @@ To upload the data, please follow these steps:
 2. Run the :ref:`data gathering script <gathering_of_results>`. The output of this step should be a compressed folder named ``results_data.zip``.
 3. Upload the compressed results folder to zenodo.
 
-  a) Go to `zenodo <https://zenodo.org/communities/openfreeenergy/records?q=&l=list&p=1&s=10&sort=newest>`_ to upload the data to the **Open Free Energy Zenodo Community**.
-  b) Click the `New upload` button on the top right.
-  c) Fill out all the relevant information. In the `Description` section, please report the OpenFE version that was used. You can get this information by running ``openfe --version`` in the commandline within the environment that you used to run the setup, simulations, and analysis of the data. In addition to that, please reference the `Industry Benchmark repo <https://github.com/OpenFreeEnergy/IndustryBenchmarks2024>`_ as
-     it contains all necessary information about the simulation details.
-  d) Add the compressed results folder (``results_data.zip``) to the zenodo upload.
-  e) Click the `Submit for review` button on the top right. We will then review the upload and publish it.
+  a) Login to zenodo or sign up if you do not have an account.
+  b) Upload the data to the **Open Free Energy Zenodo Community**. You can find the OpenFE community here: `https://zenodo.org/communities/openfreeenergy/records?q=&l=list&p=1&s=10&sort=newest <https://zenodo.org/communities/openfreeenergy/records?q=&l=list&p=1&s=10&sort=newest>`_.
+  c) Click the `New upload` button on the top right.
+  d) Fill out all the relevant information.
+
+     * Digital Object Identifier: Click `No` and `Generate a DOI now!`
+     * Resource type: Dataset
+     * Title: For example something like: OpenFE 2024 Industry Benchmarking Project, Results from X (where X could be company name, or dataset name, or author.
+     * Creators: Please add as authors everyone involved in running the calculations.
+     * Description: Please report the OpenFE version that was used. You can get this information by running ``openfe --version`` in the commandline within the environment that you used to run the setup, simulations, and analysis of the data. In addition to that, please reference the `Industry Benchmark repo <https://github.com/OpenFreeEnergy/IndustryBenchmarks2024>`_ as
+       it contains all necessary information about the simulation details.
+       E.g.: OpenFE 2024 Industry Benchmarking Project, Results from systems `jacs set; bace` and `merck set; shp2`. OpenFE version v1.0.1.  Simulation details can be found in the GitHub repository `https://github.com/OpenFreeEnergy/IndustryBenchmarks2024`
+
+  e) Add the compressed results folder (``results_data.zip``) to the zenodo upload. If multiple datasets are submitted in the same zenodo upload, please rename this file to reflect the benchmark system, e.g. `results_data_jacs_bace.zip`
```

Comment:
cc @jthorton - do we want to do this? i.e. will it be a problem for the analysis script?